pkg/topolvm-controller/cmd/run.go

```diff
@@ -76,6 +78,7 @@ func subMain() error {
 	wh := mgr.GetWebhookServer()
 	wh.Register("/pod/mutate", hook.PodMutator(mgr.GetClient(), mgr.GetAPIReader(), dec))
 	wh.Register("/pvc/mutate", hook.PVCMutator(mgr.GetClient(), mgr.GetAPIReader(), dec))
+	mgr.AddReadyzCheck("webhook", wh.StartedChecker())
```

Comment:
Please check the returned value of `AddReadyzCheck` function.

Reply:
StartedChecker return a healthz.Checker:
- https://github.com/kubernetes-sigs/controller-runtime/blob/v0.11.1/pkg/webhook/server.go#L291-L317
- https://github.com/kubernetes-sigs/controller-runtime/blob/v0.11.1/pkg/healthz/healthz.go#L183-L184